sql/sql_yacc.yy

```diff
@@ -8808,6 +8806,7 @@ table_expression:
           opt_group_clause
           opt_having_clause
           opt_window_clause
+          opt_for_system_time_clause
```

Comment:
Why is it better to put `opt_for_system_time_clause` here? I think standard isn't clear regarding this, doesn't contain examples with more complicated queries.